usr/iscsid.c

```diff
@@ -310,7 +310,8 @@ static void iscsid_shutdown(void)
 
 static void catch_signal(int signo)
 {
-	log_debug(1, "pid %d caught signal %d", getpid(), signo);
+	if (signo != SIGPIPE)
+		log_debug(1, "pid %d caught signal %d", getpid(), signo);
```

Comment:
I'd like to see a simple comment added here, e.g. "do not try to call log_debug() if there is a PIPE error because ..."

Reply:
Done.